executor/executor.cc

```diff
@@ -1233,25 +1239,23 @@ void copyin(char* addr, uint64 val, uint64 size, uint64 bf, uint64 bf_off, uint6
 
 bool copyout(char* addr, uint64 size, uint64* res)
 {
-	bool ok = false;
-	NONFAILING(
+	return NONFAILING(
 	    switch (size) {
 		    case 1:
-			    *res = *(uint8*)addr;
+			    *res = *(uint8_t*)addr;
 			    break;
 		    case 2:
-			    *res = *(uint16*)addr;
+			    *res = *(uint16_t*)addr;
 			    break;
 		    case 4:
-			    *res = *(uint32*)addr;
+			    *res = *(uint32_t*)addr;
 			    break;
 		    case 8:
-			    *res = *(uint64*)addr;
+			    *res = *(uint64_t*)addr;
```

Comment:
Are these changes in copyout required? Why? We use uint8/16/32/64 in executor.cc in other places a lot.

Reply:
I've made the change.